test/e2e/support/custom.ts

```diff
@@ -17,6 +17,13 @@ Cypress.Commands.add('dsUncheck', { prevSubject: true }, (subject: JQueryWithSel
   }
 });
 
+Cypress.Commands.add('dsSelect', { prevSubject: true }, (subject: JQueryWithSelector | undefined, name) => {
+  cy.wrap(subject).click();
+  cy.wrap(subject).parents('[data-testid="select-root"]').findByRole('option', { name }).click();
+  cy.get('body').click();
+  cy.wrap(subject);
```

Comment:
The body click is to call blur (due to delayed save state) and the last wrap is to make it chainable again, otherwise it would be the body that would get the chained operations